charts/stable/clamav/Chart.yaml

```diff
@@ -22,7 +22,7 @@ sources:
   - https://hub.docker.com/r/clamav/clamav
   - https://docs.clamav.net/
 type: application
-version: 5.0.8
+version: 5.1.0
```

Comment:
this is not a major (new feature) change ;-)

Reply:
I just wanted people to notice haha